### ISSUE TITLE
Developer API: Provide notifications (webhooks) of de-listed families?

### ISSUE THREAD

**Issue by davelab6** [google/fonts#1314]
The Developer API serves data on the families listed in the fonts.google.com directory; however, as we de-list older deprecated families (like Droid in #1223) then then Developer API also de-lists them. 

We should consider providing notifications (webhooks) of de-listed families, or move them to a tail end root node section of the returned dictionary that the developer API returns.

**Comment by Siyfion** [comment]
@davelab6 Yup, this is exactly the issue I was referring too. Sorry I wasn't a bit clearer, I had assumed that as you mentioned the Early Access Program that the font was being dropped as it wasn't graduating, but as you correctly said it was in-fact in the main directory.

The problem is that we store the font family name and use the Developer API to retrieve the most up to date URI from which to fetch (with some caching). If a font is removed like this, we have no way of knowing until we attempt to retrieve the font, at which point, it's just "gone".

**Comment by davelab6** [comment]
Would a "b sides" section of the developer api json help? 

It's possible that early access graduation rates may accelerate in the future, and also that some low usage, low quality families in the main API are delisted too.
